cmd/assetsvc/pkg/utils/postgresql_utils.go

```diff
@@ -44,34 +43,22 @@ func (m *PostgresAssetManager) GetAllChartCategories(cq ChartQuery) ([]*models.C
 	return chartsCategories, nil
 }
 
-func (m *PostgresAssetManager) GetPaginatedChartList(whereQuery string, whereQueryParams []interface{}, pageNumber, pageSize int) ([]*models.Chart, int, error) {
-	// Default (pageNumber,pageSize) = (1, 0) as in the handler.go
-	if pageNumber <= 0 {
-		pageNumber = 1
-	}
-
+func (m *PostgresAssetManager) GetPaginatedChartList(whereQuery string, whereQueryParams []interface{}, startItemNumber, pageSize int) ([]*models.Chart, error) {
 	paginationClause := ""
 	if pageSize > 0 {
-		offset := (pageNumber - 1) * pageSize
-		paginationClause = fmt.Sprintf("LIMIT %d OFFSET %d", pageSize, offset)
+		paginationClause = fmt.Sprintf("LIMIT %d", pageSize)
```

Comment:
Each time I come across this file I wish we had [an ORM](https://pg.uptrace.dev/queries/#writing-queries)... 😢 or even moving towards a single-database model across plugins.

Anyway, for the scope of this PR, +1 to the change of course!